benchmarks/asv.conf.json

```diff
@@ -0,0 +1,160 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "probnum",
+
+    // The project's homepage
+    "project_url": "https://probabilistic-numerics.github.io/probnum/intro.html",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "https://github.com/probabilistic-numerics/probnum",
```

Comment:
I recommend setting this to `".."`, so it will use the current repository rather than cloning it from upstream. This also makes it easier for the runner to alias refs to a shorter name, e.g. `refs/pull/N/head → pull/N`.

Reply:
Done.